lib/pltr/v2/reducers/books.js

```diff
@@ -42,6 +42,7 @@ const books =
           allIds: [...state.allIds, action.newBookId],
           [action.newBookId]: {
             ...action.book,
+            ...action.attributes,
```

Comment:
You'll want to specify each attribute here and compute the `newBookId` just above the return statement.